### ISSUE TITLE
question about my  own dataset

### ISSUE THREAD

**Issue by YunLixiang** [CuiRuikai/Partial2Complete#22]

**Comment by CuiRuikai** [comment]
No, we only evaluated on point clouds that are much sparser than yours. That is 2048 points. If you need to generate dense point cloud, you have to design your own decoder as ours is just a simple MLP. Scaling the MLP to millions of points will take too much memory and computation.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by YunLixiang** [comment]
Perhaps I should first make an affine transformation of point cloud data to narrow down my scene, and then conduct relevant tests. If there is a better memory computing network in the future, my data can be applied to actual production. Thank you very much for your reply, which is of great significance to a beginner of point cloud deep learning

**Comment by CuiRuikai** [comment]
It would be feasible if you first downsample the point cloud and generate only a coarse shape, then use some upsampling methods to recover the full point cloud.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
